gradle-tooling/src/main/java/com/liferay/blade/gradle/tooling/DefaultModel.java

```diff
@@ -40,6 +41,19 @@ public DefaultModel(
 		_dockerImageLiferay = dockerImageLiferay;
 		_dockerImageId = dockerImageId;
 		_dockerContainerId = dockerContainerId;
+		_appServerTomcatVersion = appServerTomcatVersion;
```

Comment:
_appServerTomcatVersion
_bundleUrl
_targetPlatformVersion should just be loaded from gradle.properties directly not invoked from tooling model.